org.testeditor.web.backend.persistence/src/test/java/org/testeditor/web/backend/persistence/CancelledTestExecutionDetailsTest.xtend

```diff
@@ -14,7 +14,7 @@ import javax.ws.rs.core.MediaType
 import org.eclipse.jgit.api.Git
 import org.eclipse.xtend.lib.annotations.FinalFieldsConstructor
 import org.junit.Test
-import org.testeditor.web.backend.persistence.AbstractPersistenceIntegrationTest
```

Comment:
Reconsider moving this file to persistence ?

Reply:
This is actually an integration test that needs to be in the persistence. I renamed the class, accordingly.

The imports do look confusing in the diff, but are actually correct: the import from *.persistence is not necessary anymore, as this is the local package now, whereas dependencies on *testexecution now need to be imported, as they are not in the same package, anymore.